cognee/tests/unit/integration/test_model_to_graph_to_model.py

```
@@ -0,0 +1,173 @@
+            "issued_by": "PU Vrsac",
+            "issued_on": "2025-11-06",
+            "number": "1234567890",
+            "expires_on": "2025-11-06",
+        },
+    )
+    nodes, edges = get_graph_from_model(boris)
+
+    try:
+        car, person = nodes[0], nodes[1]
+        edge = edges[0]
+    except:
```

Ruff

Do not use bare `except`

(E722)